src/CodiceFiscale/Subject.php

```diff
@@ -53,7 +58,7 @@ public function __construct($properties)
     /**
      * Get the name.
      *
-     * @return Returns the name.
+     * @return string Returns the name.
```

Comment:
You could remove Returns here and use an uppercase **T** for the initial **the**.